### PR TITLE
Babylon split for monorepo

### PR DESCRIPTION
**Pull request by haozhao-aws**
### What
Split the single package to three packages. This PR mainly focuses on the Core package and BabylonJS package.

**NOTE** npm publishing, README for packages, Three.js changes will not be modified or ensured to be correct from this change. Not all tests are passed. Currently only core and babylonjs package passed the tests.

### Why
We would like to decouple the packages so that we can easily control and manage each piece.

### How was the change tested? Please list exact steps taken to manually test this change.
`npm install` =>
* `npm run --workspace=./packages/amazon-sumerian-hosts-core build-test`
* `npm run --workspace=./packages/amazon-sumerian-hosts-babylon build-test`
* `npm run --workspace=./packages/amazon-sumerian-hosts-babylon start` =>
  * Verified examples worked as expected
* `npm pack --workspace=./packages/amazon-sumerian-hosts-core —dry-run`
* `npm pack --workspace=./packages/amazon-sumerian-hosts-babylon --dry-run`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amazon-sumerian-hosts-core/src/core/FeatureDependentInterface.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 /* eslint-disable no-unused-vars */
 /* eslint-disable max-classes-per-file */
-import HostObject from 'app/HostObject';
+import HostObject from 'core/HostObject';
```
Comment (brhook-aws) [inline packages/amazon-sumerian-hosts-core/src/core/FeatureDependentInterface.js]: We make this change in some files, and not others. Why did we switch?

**Review by brhook-aws — APPROVED** [review]
Please have at least one other person review.

**Review by brhook-aws — CHANGES_REQUESTED** [review]
We cannot merge into the branch [aws-samples:master](https://github.com/aws-samples/amazon-sumerian-hosts/tree/master). This branch is used for our GitHub Pages. Please make a new branch.

File: packages/amazon-sumerian-hosts-babylon/src/Babylon.js/awspack/Speech.js [hunk 2]
```diff
@@ -1,63 +1,61 @@
...
-    this._audio.stop();
-
-    AbstractSpeech.prototype.stop.call(this);
-  }
-}
-
-export default Speech;
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import {Speech as CoreSpeech, AbstractSpeech} from '@amazon-sumerian-hosts/core';
```
[the remark this answers is not in the document]
Comment (haozhao-aws) [reply packages/amazon-sumerian-hosts-babylon/src/Babylon.js/awspack/Speech.js]: Good catch. Refactored.

File: packages/amazon-sumerian-hosts-babylon/src/Babylon.js/awspack/TextToSpeechFeature.js [hunk 3]
```diff
@@ -1,95 +1,100 @@
...
+
+          if (this._attachTo !== undefined && !this._isGlobal) {
+            result.audio.attachToMesh(this._attachTo);
+          }
+        });
+      });
+  }
+
+   _createSpeech(text, speechmarks, audioConfig) {
+    return new Speech(this, text, speechmarks, audioConfig);
```
[the remark this answers is not in the document]
Comment (haozhao-aws) [reply packages/amazon-sumerian-hosts-babylon/src/Babylon.js/awspack/TextToSpeechFeature.js]: Refactored

File: jsdoc.conf.json [hunk 4]
```diff
@@ -33,6 +33,6 @@
     "readme": "./docs/readme.md"
   },
   "source": {
-    "include": ["./src/core/", "./src/three.js", "./src/Babylon.js"]
+    "include": ["./packages/amazon-sumerian-hosts-core/src/core/", "./packages/amazon-sumerian-hosts-core/src/three.js", "./packages/amazon-sumerian-hosts-core/src/Babylon.js"]
```
[the remark this answers is not in the document]
Comment (haozhao-aws) [reply jsdoc.conf.json:36]: Updated

File: package.json [hunk 5]
```diff
@@ -1,31 +1,24 @@
...
+    "release": "npm run --workspaces build",
     "docs": "jsdoc -c jsdoc.conf.json"
   },
-  "keywords": [],
-  "author": "",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/aws-samples/amazon-sumerian-hosts.git"
-  },
```
[the remark this answers is not in the document]
Comment (haozhao-aws) [reply package.json:20]: Added in all package.json files.

File: packages/amazon-sumerian-hosts-babylon/src/Babylon.js/animpack/index.js [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  RandomAnimationState,
+  Blend1dState,
+  Blend2dState,
+  AnimationLayer,
+  LayerBlendModes,
+  DefaultLayerBlendMode } from '@amazon-sumerian-hosts/core';
+import AnimationFeature, {AnimationTypes} from './AnimationFeature';
+import SingleState from './state/SingleState';
+
+// /**
```
[the remark this answers is not in the document]
Comment (haozhao-aws) [reply packages/amazon-sumerian-hosts-babylon/src/Babylon.js/animpack/index.js]: Removed

**Comment by haozhao-aws** [comment]
> We cannot merge into the branch [aws-samples:master](https://github.com/aws-samples/amazon-sumerian-hosts/tree/master). This branch is used for our GitHub Pages. Please make a new branch.

Has changed to mainline.

**Review by brhook-aws — APPROVED** [review]

**Pull request merged**